### PR TITLE
Changelog update for October out-of-band patch releases

### PR DESCRIPTION
**Pull request by pkprzekwas**
**What type of PR is this?**
/kind chore

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: docs/changelogs/CHANGELOG-2.21.md [hunk 1]
```diff
@@ -14,6 +14,17 @@
...
+
+## [v2.21.14](https://github.com/kubermatic/kubermatic/releases/tag/v2.21.14)
+
+### Bugfixes
+
+- Increase default CPU limits for KKP API/seed/master-controller-managers to prevent general slowness ([#12771](https://github.com/kubermatic/kubermatic/pull/12771))
+
+### Updates
+
+- Add support for Cilium 1.12.15, mitigating an high CVE-2023-44487 ([#12767](https://github.com/kubermatic/kubermatic/pull/12767))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.21.md]: `a high-severity vulnerability, CVE-2023-44487`

File: docs/changelogs/CHANGELOG-2.23.md [hunk 2]
```diff
@@ -6,6 +6,19 @@
...
+
+### Bugfixes
+
+- Fix Digitalocean CSI addon failing to render ([#12739](https://github.com/kubermatic/kubermatic/pull/12739))
+- Fix node-labeller controller not applying the `x-kubernetes.io/distribution` label to RHEL nodes ([#12751](https://github.com/kubermatic/kubermatic/pull/12751))
+- Increase default CPU limits for KKP API/seed/master-controller-managers to prevent general slowness ([#12764](https://github.com/kubermatic/kubermatic/pull/12764))
+
+### Updates
+
+- Add support for Cilium 1.13.8, mitigating an high CVE-2023-44487 ([#12762](https://github.com/kubermatic/kubermatic/pull/12762))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.23.md]: Same typo

File: docs/changelogs/CHANGELOG-2.21.md (hunk 1 above)
Comment (pkprzekwas) [reply docs/changelogs/CHANGELOG-2.21.md]: Damn, the release is out. Let me fix this where possible.

Comment (xrstf) [reply docs/changelogs/CHANGELOG-2.21.md]: It's just nitpicking, don't stress too much :)

Comment (pkprzekwas) [reply docs/changelogs/CHANGELOG-2.21.md]: Good catch. I've updated the text wherever it was simple.

**Pull request merged**